forge-gui/res/cardsfolder/upcoming/bane_lord_of_darkness.txt

```diff
@@ -0,0 +1,13 @@
+Name:Bane, Lord of Darkness
+ManaCost:1 W U B
+Types:Legendary Creature God
+PT:5/2
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | CheckSVar$ CurrentLife | SVarCompare$ LEX | Description$ As long as your life total is less than or equal to half your starting life total, CARDNAME has indestructible.
+SVar:CurrentLife:Count$YourLifeTotal
+SVar:X:Count$YourStartingLife/HalfDown
+T:Mode$ ChangesZone | ValidCard$ Creature.nonToken+Other+YouCtrl | Origin$ Battlefield | Destination$ Graveyard | Execute$ DBAskOpponentDrawOrPlay | TriggerZones$ Battlefield | TriggerDescription$ Whenever another nontoken creature you control dies, target opponent may have you draw a card. If they don't, you may put a creature card with equal or lesser toughness from your hand onto the battlefield.
+SVar:DBAskOpponentDrawOrPlay:DB$ GenericChoice | ValidTgts$ Opponent | Choices$ DBDrawCard,DBCheatCreature
```

Comment:
How does the AI handle this choice?

Reply:
The AI seems to always be picking the first option (drawing). Even when my hand was empty (and option 2 is therefore much better since I won't have anything to play), the AI still makes me draw.

Reply:
AI might be clean up later by me or @Agetian